monitor_check.py

```diff
@@ -279,6 +335,52 @@ def bug(bugs, package):
 
 counter = Counter()
 
+def pkgname(srcname):
+    pkgs = repoquery(source=srcname.rsplit("-", 2)[0], version="rawhide")
+    for pkg in pkgs:  # a only gets evaluated here
+    #    if pkg.reponame == "fedorarawhide":
+        return pkg.source_name
```

Comment:
You might want to do something like.

```suggestion
        return pkg.source_name
    raise RuntimeError(f"Cannot find source for {name}. Hint: Remove the cache in {DNF_CACHEDIR}")
```